core/layout_api.php

```diff
@@ -256,7 +259,8 @@ function layout_head_css() {
 		html_css_cdn_link( 'https://maxcdn.bootstrapcdn.com/font-awesome/' . FONT_AWESOME_VERSION . '/css/font-awesome.min.css' );
 
 		# theme text fonts
-		html_css_cdn_link( 'https://fonts.googleapis.com/css?family=Open+Sans:300,400' );
+		$t_font_family =  config_get( 'font_family', null, null, ALL_PROJECTS );
```

Comment:
I thought about this initially but decided against it. It complicates the logic since `layout_user_font_preference()` has to produce different inline style element then 'config_get()' has to include the correct font files in both CDN and non-CDN cases. This complexity is rather unnecessary.

If any of the scenarios you mentioned takes place, impacted users will their default browser font (most likely ' 'sans serif'). They can simply fix the selection in their own preferences form.